### DISCUSSION TITLE
Wazuh - Agent register

### DISCUSSION THREAD

**Discussion by b4b857f6ee** [Security-Onion-Solutions/securityonion#7417] [Unsupported Versions] [answered]
Hello,

I'm on the last version 20220203

I'm using a powershell script to register and install the wazuh agent on Windows and for some reason when i run the script first i got the agent but with "any"...
![image](https://user-images.githubusercontent.com/36012515/156342895-2067377f-ae74-416b-937d-2564766a21ab.png)

And if i run it a second time it's register with the proper IP...
I do not understand why..

Any idea?

The script (the end)

![image](https://user-images.githubusercontent.com/36012515/156342841-a1dd9d16-9058-45b4-ba6f-1ae3cc2412d2.png)

**Comment by b4b857f6ee** [comment, marked as answer]
Does anyone find in the docker this file?  From this answer : https://github.com/wazuh/wazuh/issues/12484#issuecomment-1057190612

Manager side
Set <use_source_ip> to yes at etc/ossec-conf:

<auth>
  <use_source_ip>no</use_source_ip>
</auth>

**Reply by sleepingbel** [reply]
Hello,

It is not in the docker but on the manager on location
/opt/so/conf/wazuh/ossec.conf

This is the manual page for it [SO Wazuh](https://docs.securityonion.net/en/2.3/wazuh.html?highlight=wazuh)
